webapp/src/webapp_globals.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export const {
```

Comment:
You need to defined the types like how its done in user survey plugin for example - https://github.com/mattermost/mattermost-plugin-user-survey/blob/29685cc4d7ca0a2e1de196c23f6165caeacaf1b4/webapp/src/types/mattermost-webapp/index.d.ts#L10

Reply:
Done.